packages/sirius-web/backend/sirius-web-tests-data/src/main/resources/sirius-web-scripts/papaya-library.sql

```diff
@@ -572,4 +572,92 @@ INSERT INTO library (
   'The Sirius Web Tests Data library',
   '2025-03-14 12:25:11.524',
   '2025-03-14 12:25:11.524'
+);
+
+INSERT INTO semantic_data (
```

Comment:
I added a new version of the library without the annotation to be able to trigger an update that could generate proxies.